<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>746</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>QtPlaskin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Overview</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="spacing">
               <number>25</number>
              </property>
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>20</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>20</number>
              </property>
              <item>
               <widget class="QTableWidget" name="condList">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::SingleSelection</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="showGrid">
                 <bool>false</bool>
                </property>
                <property name="sortingEnabled">
                 <bool>true</bool>
                </property>
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>40</number>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderDefaultSectionSize">
                 <number>20</number>
                </attribute>
                <attribute name="verticalHeaderMinimumSectionSize">
                 <number>10</number>
                </attribute>
                <column>
                 <property name="text">
                  <string>#</string>
                 </property>
                 <property name="foreground">
                  <brush brushstyle="NoBrush">
                   <color alpha="255">
                    <red>109</red>
                    <green>109</green>
                    <blue>109</blue>
                   </color>
                  </brush>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Condition</string>
                 </property>
                </column>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="condButton">
                <property name="text">
                 <string>Plot</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="ConditionsPlotWidget" name="condWidget" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Densities</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>25</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <widget class="QTableWidget" name="speciesList">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>40</number>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>20</number>
              </attribute>
              <attribute name="verticalHeaderMinimumSectionSize">
               <number>10</number>
              </attribute>
              <column>
               <property name="text">
                <string>#</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Species</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="plotButton">
              <property name="text">
               <string>Plot</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="DensityPlotWidget" name="densWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Reactions</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>25</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <widget class="QTableWidget" name="reactList">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>40</number>
              </attribute>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>20</number>
              </attribute>
              <attribute name="verticalHeaderMinimumSectionSize">
               <number>10</number>
              </attribute>
              <column>
               <property name="text">
                <string>#</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Reaction</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="reactButton">
              <property name="text">
               <string>Plot</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="RatePlotWidget" name="reactWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Sensitivity analysis</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>15</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <widget class="QTableWidget" name="speciesSourceList">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>40</number>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>20</number>
              </attribute>
              <attribute name="verticalHeaderMinimumSectionSize">
               <number>10</number>
              </attribute>
              <column>
               <property name="text">
                <string>#</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Species</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="Combo_filter">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string>Filter at 10%</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Filter at 1%</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Filter at 0.1%</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Filter at 0.01%</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Show all</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sourceButton">
              <property name="text">
               <string>Plot</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="SourcePlotWidget" name="sourceWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>QtPlaskin - (c) Alejandro Luque, Instituto de Astrofísica de Andalucía (IAA), CSIC, 2011</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1276</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionImport_from_directory"/>
    <addaction name="actionExport_data"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionUpdate"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionLog_scale_in_time"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLog_scale_in_time">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log scale in time</string>
   </property>
  </action>
  <action name="actionExport_data">
   <property name="text">
    <string>Export data...</string>
   </property>
  </action>
  <action name="actionStart_a_simulation">
   <property name="text">
    <string>Start a simulation...</string>
   </property>
  </action>
  <action name="actionImport_from_directory">
   <property name="text">
    <string>Import from directory...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionFilter_small_rates">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter small rates</string>
   </property>
   <property name="toolTip">
    <string>When checked, some rates are not displayed in the sensitivity analisys to avoid cluttering</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DensityPlotWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SourcePlotWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RatePlotWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConditionsPlotWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
